templates/bundles/DemosPlanStatementBundle/DemosPlanAssessment/view_statement.html.twig

```diff
@@ -1,5 +1,19 @@
 {% extends '@DemosPlanCore/DemosPlanCore/base.html.twig' %}
 
+{# if there are statement votes ("Mitzeichner"), then the option "Schlussmitteilung versenden" plus corresponding hint will be shown #}
+{% set templateVars.sendFinalEmail = true %}
+{% set templateVars.finalEmailOnlyToVoters = false %}
+{% if templateVars.table.statement.feedback == 'snailmail' %}
+  {% set templateVars.table.statement.feedback = "via.post"|trans %}
+  {% if templateVars.table.statement.votes is empty %}
+    {% set templateVars.sendFinalEmail = false %}
+  {% else %}
+    {% set templateVars.finalEmailOnlyToVoters = true %}
+  {% endif %}
+{% elseif templateVars.table.statement.feedback == 'email' %}
+    {% set templateVars.table.statement.feedback = "via.email"|trans %}
+{% endif %}
```

Comment:
I don't quite understand why this has to happen in Twig. We want to reduce the amount of Twig templating overall and move towards Vue. Yes this is simple logic, but still, logic in Twig is less readable and less performant than logic in PHP.

Reply:
See commit message.